.rubocop.yml

```diff
@@ -16,4 +16,6 @@ AllCops:
     - 'spec/rails_helper.rb'
     -  'spec/spec_helper.rb'
 Metrics/BlockLength:
-  ExcludedMethods: ['describe', 'context', 'feature', 'scenario', 'let']
\ No newline at end of file
+  ExcludedMethods: ['describe', 'context', 'feature', 'scenario', 'let']
+Style/AlignHash:
```

Comment:
Tem algum motivo específico pra desabilitar?

Reply:
Aparece o erro no rubocop "Align the elements of a hash literal if they span more than one line", Tentei alinhar mas não obtive sucesso. Parti para o google e a "solução" que encontrei foi adicionar:
`Style/AlignHash:
  Enabled: false`